hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -32,6 +32,14 @@
     }
 }
 
+MAYA_GPU_ITEM_TYPE_SETTINGS = {
+    "maya.geometry": {
+        "publish_type": "GPU Alembic Cache",
```

Comment:
I'm not sure if I'm remembering right, but please make sure the publish type is consistent (GPU Cache vs. GPU Alembic Cache) everywhere (i.e. the sgtk_config_environments file)

Reply:
sure.
Made it consistent across as "Publish GPU Alembic Cache"